routingapi/routingapi_test.go

```diff
@@ -72,12 +72,13 @@ var _ = Describe("Routing API", func() {
 				Expect(client.RouterGroupWithNameArgsForCall(0)).To(Equal("my-router-group"))
 
 				expectedTTL := 42
+				hostTLSPort := -1
 				expectedRouteMapping := models.TcpRouteMapping{TcpMappingEntity: models.TcpMappingEntity{
 					RouterGroupGuid: "router-group-guid",
 					HostPort:        1234,
 					ExternalPort:    5678,
 					HostIP:          "myhost",
-					HostTLSPort:     -1,
+					HostTLSPort:     &hostTLSPort,
```

Comment:
This should be an int, not pointer to int.. Something seems fishy. Did the tcpmappingentity model not get updated?